tensorrt_llm/_torch/pyexecutor/py_executor_creator.py

```diff
@@ -306,6 +308,40 @@ def create_py_executor(executor_config: ExecutorConfig,
             decoder = TorchDecoder(
                 max_seq_len=model_engine.max_seq_len,
                 mixed_decoder=pytorch_backend_config.mixed_decoder)
+
+    if lora_config is not None:
+        from tensorrt_llm.bindings import LoraModule
+        load_torch_hf_lora(lora_config)
+        model_binding_config = model_engine.model.model_config.get_bindings_model_config(
+        )
+        lora_modules = LoraModule.create_lora_modules(
+            lora_config.lora_target_modules, model_binding_config.hidden_size,
+            model_binding_config.mlp_hidden_size,
+            model_binding_config.num_heads, model_binding_config.num_heads,
+            model_binding_config.head_size)
+        model_binding_config.use_lora_plugin = True
+        model_binding_config.lora_modules = lora_modules
+        model_binding_config.max_lora_rank = lora_config.max_lora_rank
+
+        max_lora_rank = lora_config.max_lora_rank
+        num_lora_modules = model_engine.model.model_config.pretrained_config.num_hidden_layers * \
+            len(lora_config.lora_target_modules + lora_config.missing_qkv_modules)
+
+        # TODO smor- need to figure out how to set these values
+        max_loras = 4
+        max_cpu_loras = 4
+        executor_config.peft_cache_config = tllm.executor.PeftCacheConfig(
```

Comment:
I remember this is used to determine the cache size and hence how many cache you can store.